api/src/main/java/org/openmrs/util/DatabaseUpdater.java

```diff
@@ -68,7 +68,7 @@
 	
 	private static final Log log = LogFactory.getLog(DatabaseUpdater.class);
 	
-	private static final String CHANGE_LOG_FILE = "liquibase-update-to-latest.xml";
+	private static final String CHANGE_LOG_FILE = "liquibase-master.xml";
```

Comment:
I have updated the ticket description, we don't need to rename the `liquibase-update-to-lastest.xml` fie